operations/patch.go

```diff
@@ -89,8 +94,8 @@ func Patch() cli.Command {
 			setPlainLogger,
 			mutuallyExclusiveArgs(false, patchDescriptionFlagName, autoDescriptionFlag),
 			mutuallyExclusiveArgs(false, preserveCommitsFlag, uncommittedChangesFlag),
-			mutuallyExclusiveArgs(false, repeatDefinitionFlag, repeatPatchIdFlag,
-				repeatFailedDefinitionFlag),
+			mutuallyExclusiveArgs(false, repeatDefinitionFlag, repeatPatchIdFlag, repeatFailedDefinitionFlag),
+			conditionallyRequiredArgs(jsonFlagName, skipConfirmFlagName),
```

Comment:
I don't feel too strongly about this but could we just enforce this ourselves? I think users would expect confirmation to be skipped, and we can mention it in the help text. Code-wise we can just set skipConfirm to true if the json flag is set so it won't change much.

Reply:
Hm that's true, my thinking was that if we do do that, we can't inform the user that we have done so. I'll switch it to silently set it since I agree that it would be an expectation!